webapp/src/Components/Shared/Selects/PlaylistSelect.tsx

```diff
@@ -272,35 +303,45 @@ class PlaylistSelectComponent extends React.PureComponent<Props, State> {
                 label="Show only ranked playlists"
             />
 
-        return (
-            <ExpansionPanel square={false} expanded={this.state.optionsExpanded}>
-                <ExpansionPanelSummary
-                    classes={classes}
-                    expandIcon={
-                        <IconButton onClick={this.handleExpandedChange}>
-                            <Tooltip title="Playlist options">
-                                <ExpandMore/>
-                            </Tooltip>
-                        </IconButton>
-                    }
-                >
-                    {playlistsMultiSelect}
-                </ExpansionPanelSummary>
-                <ExpansionPanelDetails className={classes.panelDetails}>
-                    {filterCurrentCheckbox}
-                    {filterStandardCheckbox}
-                    {filterRankedCheckbox}
-                </ExpansionPanelDetails>
-            </ExpansionPanel>
+        return (<>
+
+                {!this.props.justDropdown &&
+                <ExpansionPanel square={false} expanded={this.state.optionsExpanded}>
+                    <ExpansionPanelSummary
+                        classes={classes}
+                        expandIcon={
+                            <IconButton onClick={this.handleExpandedChange}>
+                                <Tooltip title="Playlist options">
+                                    <ExpandMore/>
+                                </Tooltip>
+                            </IconButton>
+                        }
+                    >
+                        {playlistsMultiSelect}
+                    </ExpansionPanelSummary>
+                    <ExpansionPanelDetails className={classes.panelDetails}>
+                        {filterCurrentCheckbox}
+                        {filterStandardCheckbox}
+                        {filterRankedCheckbox}
+                    </ExpansionPanelDetails>
+                </ExpansionPanel>
+
+                }
+                {
+                    this.props.justDropdown && <>{playlistsMultiSelect}</>
```

Comment:
Delete the blank line above, and collapse this bracket to a single line. `<>{playlistsMultiSelect}</>` should be shortenable to `playlistsMultiSelect`.